crates/pera-mpc-types/src/lib.rs

```diff
@@ -0,0 +1,41 @@
+use crypto_bigint::Uint;
+use serde::{Deserialize, Serialize};
+
+// Todo (#369): Change types to real types once the class groups keygen is ready and doesn't take forever
+pub type MockProof = Vec<u8>;
+pub type ClassGroupsPublicKeyAndProofBytes = Vec<u8>;
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub struct ClassGroupsPublicKeyAndProof(
+    class_groups::CompactIbqf<{ class_groups::SECRET_KEY_SHARE_DISCRIMINANT_LIMBS }>,
```

Comment:
I'll name them, but I don't think we should separate them. This is the proof for this specific class-groups key pair. Additionally, we need to access them as a mapping of authority to encryption key & proof, since each validator holds this map and passes it as part of the DKG public Input.